posthog-node/src/posthog-node.ts

```diff
@@ -114,12 +113,20 @@ export class PostHog extends PostHogCoreStateless implements PostHogNodeV1 {
       super.captureStateless(distinctId, event, props, { timestamp, disableGeoip, uuid })
     }
 
+    const _getFlags = (
```

Comment:
Why this abstraction? Feels like a lot more code that doesn't do anything?

Reply:
You were too fast with the review lol was just writing a letter to you xD